### PR TITLE
feat: [Profile] Add `location` field

### PR DESCRIPTION
**Pull request by damassi**
Adds a new `location` field to the `Profile` type and a little bit of alphabetization cleanup.

### CODE REVIEW

File: src/schema/v2/profile.ts [hunk 1]
```diff
@@ -63,17 +58,26 @@ export const ProfileType = new GraphQLObjectType<any, ResolverContext>({
...
     },
+    isPubliclyVisible: {
+      type: GraphQLBoolean,
+      resolve: (profile) => profile && profile.published && !profile.private,
+    },
     isPublished: {
       type: GraphQLBoolean,
       resolve: ({ published }) => published,
     },
+    location: {
```
Comment (damassi) [inline src/schema/v2/profile.ts:69]: Only addition -- the rest is just alphabetization for better readability

File: src/schema/v2/profile.ts [hunk 2]
```diff
@@ -63,17 +58,26 @@ export const ProfileType = new GraphQLObjectType<any, ResolverContext>({
...
+      type: GraphQLString,
+      resolve: ({ location }) => location,
```
Comment (dzucconi) [inline src/schema/v2/profile.ts:71]: This isn't a proper structured location?

Comment (damassi) [reply src/schema/v2/profile.ts:71]: Not sure I follow

Comment (dzucconi) [reply src/schema/v2/profile.ts:71]: Ah OK. Both user profiles and shows have locations that are more like this: https://github.com/artsy/metaphysics/blob/master/src/schema/v2/me/myLocation.ts

**Review by dzucconi — APPROVED** [review]

**Pull request merged**